db/migrate/20220721114345_add_reverse_read_length_to_request_metadata.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# Adding reverse_read_length column to request_metadata
+class AddReverseReadLengthToRequestMetadata < ActiveRecord::Migration[6.0]
+  def change
+    add_column :request_metadata, :reverse_read_length, :integer
+  end
+end
```

Comment:
I know this was mentioned as part of the story, but @andrewsparkes has been doing a bit of work to reduce the need to add columns to the request_metadata tables. It might be worth dropping this migration for now.